einstein-ai/1.0/modules/ROOT/pages/configuring-embeddings-operations.adoc

```diff
@@ -0,0 +1,110 @@
+= Configuring Embedding Operations for Einstein AI Connector
+
+[[adhoc-file-query]]
+== Configure the Adhoc File Query Operation
+
+The *Embedding Adhoc File Query* operation takes a document and ingests it into the vector database along with its query. 
+
+The output of this operation is a set of scores with the complete content of the document that's the most likely answer to the query. The vector database is used to identify the numeric representation of the content before creating the likely score.
+
+include::connectors::partial$t-configure-operations.adoc[]
+* *Prompt* 
++
+Plain text for the prompt to send to the LLM
+include::connectors::partial$agentforce-connector-config.adoc[tags=embedding-configs]
```

Comment:
double checking this agentforce partial should be here

Reply:
Yes, this is the same for both connectors. I probably should have name the file something more generic but I don't want to break anything at this point.